guides/common/modules/ref_getting-help-with-project-api.adoc

```diff
@@ -0,0 +1,11 @@
+[id="getting-help-with-{project-context}-api"]
+= Getting help with {Project} API
+
+View the full API reference on your {ProjectServer} at the following URL:
+
+[source, none, options="nowrap", subs="+quotes,attributes"]
+----
+https://_{foreman-example-com}_/apidoc/v2.html
```

Comment:
I'd use just `/apidoc/` without the `v2.html`, both give the same content, but if we ever introduce an v3 API and make it default it's one place less to update :)

Reply:
I opened https://github.com/theforeman/foreman-documentation/pull/3863 to get this updated in all versions